source/validate_id.cpp

```diff
@@ -1216,12 +1216,11 @@ bool idUsage::isValid<SpvOpFunctionParameter>(const spv_instruction_t* inst,
                                    << inst->words[resultTypeIndex]
                                    << "' is not defined.";
```

Comment:
That was my thought. I didn't want to break all of the ValidateID unit tests in this commit. That is something I have been prototyping in another branch.